src/index.css

```diff
@@ -1,3 +1,9 @@
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@300&display=swap');
```

Comment:
import the complete family major weights like:
Regular
Medium
Semi Bold
Bold
Extra Bold

Reply:
Done